src/Classes/DogDisplayer.php

```diff
@@ -26,4 +30,50 @@ public static function displayMainPage(array $dogs): string {
         }
         return $result;
     }
+
+
+    /**
+     * function to generate a long string of html containing an individual dog article for the dog display page
+     * @param Dog $dog a dog instance containing dog information
+     * @return string a string containing the dog article html
+     */
+
+    public static function displayDetailedInfo(\Fetch\Classes\Dog $dog): string {
+        $result = '<article>';
+        $result .= '<img tabindex="2" src="assets/dog-images/' . $dog->getId() . '.jpeg" alt="' . $dog->getName() .'">';
+        $result .= '<div>';
+        $result .= '<h3 tabindex="2">' . $dog->getName() . '</h3>';
+        $result .= '<p tabindex="2">Temperament: ' . $dog->getTemperament() . '</p>';
+        $result .= '<p tabindex="2">Weight: ' . $dog->getWeightMetric() . ' kg</p>';
+        if ($dog->getWeightImperial() !== '') {
+            $result .= '<p tabindex="2">Weight: ' . $dog->getWeightImperial() . ' lb</p>';
+        }
+        if ($dog->getHeightMetric() !== '') {
+            $result .= '<p tabindex="2">Height: ' . $dog->getHeightMetric() . ' cm</p>';
+        }
+        if ($dog->getHeightImperial() !== '') {
+            $result .= '<p tabindex="2">Height: ' . $dog->getHeightImperial() . '"</p>';
```

Comment:
Could we say inches instead of " to be a little clearer and match with the cm above it?